### ISSUE TITLE
Check for mimimum wcag2 and/or optimum wcag2?

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#127]
We are now checking if a website doesn't fail for the minimum requirements of WCAG2. If one of the WCAG2 techniques is available, it will pass.
There are success criteria that have a minimum technique, but more techniques is sometimes better. 
For example: bypass blocks 2.4.1 (http://www.w3.org/WAI/WCAG20/quickref/#qr-navigation-mechanisms-skip) let's users choose between two different technologies:
- Creating links to skip blocks of repeated material (skip links)
- Grouping blocks of repeated material in a way that can be skipped (block headings)

Drupal CMS for example has implemented both, but one of the techniques is sufficient. Is the other technique in that situation a suggestion, like the advised techniques?
